.golangci.yaml

```diff
@@ -170,6 +170,10 @@ issues:
     - path: test/
       linters:
         - testpackage
+    - path: pkg/local_workflows/auth|report_analytics|whoami|output_workflow.go
```

Comment:
Why are we ignoring static checks on new changes?

Reply:
In https://github.com/snyk/go-application-framework/pull/131/commits/edff805cf117356939f7b901844f3f84ed54425e, I introduced an annotation which is now caught by the linter. However I don't want to address the existing instances of `GetLogger` as part of this PR. I've captured the work in a ticket for follow up either in the next sprint(s) or as part of cool down.

Reply:
Should we maybe not use the `deprecated` label yet? It'll surface in every extension as well. So maybe we just mention to use the newer `GetEnhancedLogger()` in the docs for now and remove the linter exception.